tigerpath/majors_and_certificates/majors/NES_2018.json

```diff
@@ -0,0 +1,138 @@
+          "explanation": "All students are required to take NES 300 (Seminar in Research Methods) in the junior year unless they are studying abroad that semester, in which case they are required to take NES 300 in the senior year.",
+          "course_list": [
+            "NES 300"
+          ]
+        },
+        {
+          "name": "Distribution Requirements",
+          "max_counted": null,
+          "min_needed": null,
+          "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nSeven other deparmentals that fulfill the following distribution requirements in historical periods, regions, and disciplines.\nA single course may satisfy multiple distribution requirements across these domains. For example, a course focusing on 20th century Turkish literature may count as modern (Historical Periods), Ottoman Empire/Turkey (Regions), and literature (Disciplines).\nA single course may not, however, satisfy more than one distribution requirement within a single domain; a course on Arabic literature that spans the pre-modern and modern periods would thus not satisfy both the pre-modern and modern Historical Periods requirements.",
+          "req_list": [
+            {
+              "name": "Historical Periods",
+              "max_counted": null,
+              "min_needed": null,
+              "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nAt least one course focused on the pre-modern Near East (before 1800) and at least one course focused on the modern Near East.",
+              "no_req": null
+            },
+            {
+              "name": "Regions",
+              "max_counted": null,
+              "min_needed": null,
+              "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nAt least two regionally specific courses. Each must focus on one of the following six sub-regions of the Near East. They may not focus on the same region: Egypt, North Africa, and Andalusia; The Levant, Iraq, and the Arabian peninsula; Ottoman empire/Turkey; Iran; Muslim South and Central Asia; Diasporic communities.",
+              "no_req": null
+            },
+            {
+              "name": "Disciplines",
+              "max_counted": null,
+              "min_needed": null,
+              "explanation": "<i>Editor's Note: This requirement is too broad or requires course approval from the concentrator's Director of Undergraduate Studies, so TigerPath does not check it.</i>\n\nAt least one course in two of the following four disciplines: History, Literature, Social sciences, Religion.",
+              "no_req": null

```

Comment:
At least here, it might be possible to get a good approximation by encoding the cross-listed department.
For example, HIS for history, COM for literature, and REL for religion.
Not exactly sure what "Social sciences" entails, though.

Reply:
You probably already know, by the way, but there's a pretty comprehensive list of possible courses on the [Undergraduate Announcement](https://ua.princeton.edu/academic-units/department-near-eastern-studies) page, under the "courses" tab, and a currently offered list on the [department webpage](https://nes.princeton.edu/courses).

A search for the word "literature" in the course titles, for example, gives 2 currently offered courses (NES 387 and HEB 315), but there are several more that haven't been offered in years.